integration/samples/jsx/specs/react-jsx.ts

```diff
@@ -1,11 +1,14 @@
 import { expect } from 'chai';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as API from '../dist/esm5/jsx.js';
 
 describe(`@sample/jsx`, () => {
 
   describe(`jsx.es5.js`, () => {
+    let API;
+    before(() => {
+      API = require('../dist/esm5/jsx.js');
+    });
```

Comment:
Prevents TypeScript compilation error for spec suite. Fails `before` hook in test run, if bundle files does not exist.